lib/kafka_rest_client.js

```diff
@@ -77,7 +111,7 @@ function KafkaRestClient(options, callback) {
             /*jshint camelcase: true */
             /* eslint-enable camelcase */
         });
-    } else {
+    }else {
```

Comment:
nit: format

Reply:
This one is gone after the revision update.